e2e/cypress/integration/compose.ts

```diff
@@ -7,8 +7,8 @@ describe('Composing emails', () => {
     }
 
     function composeNew() {
-        cy.visit('/compose?new=true');
-        cy.closeWelcomeDialog();
+	cy.visit('/compose?new=true');
```

Comment:
Can we keep 4 spaces here (and in the rest of the file)? :)

Reply:
hm, while I cant see where it changed back, my checkout of the branch is consistently 4..